scripts/utils/lit.messages.js

```diff
@@ -0,0 +1,26 @@
+// FIXME: This file is completely obsolete. Ideally, we'd be holding these
```

Comment:
If we go the `res.random` route, this can turn into a JSON file with a structure like

```json
{
  "success": [
    "Sure",
    "Got you!"
  ],
  "error": [
    "You dun goofed",
    "Nah"
  ]
}
```

and we get to make this a config file that we can edit (like you were talking about) and we can also pull it into any file using `const responses = require('../lib/messages.json');` because in Node.js you can import a JSON blob as an object and it'll be parsed in automatically. 

I would also move this file into `lib` because it's really not meant to be be in scripts.